src/modules/flow/freegeoip/freegeoip.json

```diff
@@ -14,7 +14,7 @@
         "open": "freegeoip_open",
         "close": "freegeoip_close"
       },
-      "name": "location/freegeoip",
+      "name": "freegeoip/location",
```

Comment:
This would ideally be `location/freegeoip`, as there might be other providers for the same interface (GPS, GPRS antenna triangulation, other IP->location databases, etc).